android-auto-app/src/main/java/com/mapbox/maps/testapp/auto/car/CarJavaInterfaceChecker.java

```diff
@@ -1,23 +0,0 @@
-package com.mapbox.maps.testapp.auto.car;
-
-import static com.mapbox.maps.extension.androidauto.MapboxCarUtilsKt.initMapSurface;
-
-import androidx.car.app.Session;
-
-import com.mapbox.maps.MapInitOptions;
-import com.mapbox.maps.extension.androidauto.MapSurfaceReadyCallback;
-import com.mapbox.maps.extension.androidauto.OnMapScaleListener;
-import com.mapbox.maps.extension.androidauto.OnMapScrollListener;
-
-public class CarJavaInterfaceChecker {
```

Comment:
would be good to have a java class that validate the Java API surface.

Reply:
Ok sounds good, added